stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1234,6 +1249,11 @@ public final class TestSuite {
     case .parameterized(code: let code, _):
       code(parameter!)
     }
+
+#if SWIFT_RUNTIME_ENABLE_LEAK_CHECKER
```

Comment:
I don't understand the question.  The leaks checker will catch all Swift object references as long as we tell it to start up here.  There is no further plumbing.